### ISSUE TITLE
alist.sort() should be sorted(alist) in Style.py

### ISSUE THREAD

**Issue by rgacote** [python-excel/xlwt#80]
alist.sort() does not work in Python3. After digging about the net, I found several recommendations to simply use sorted(alist). This works nicely. Entering a ticket since I didn't see an open one here documenting this.

**Comment by cjw296** [comment]
Urm?

```
$ python3.4
Python 3.4.2 (v3.4.2:ab2c023a9432, Oct  5 2014, 20:42:22) 
[GCC 4.2.1 (Apple Inc. build 5666) (dot 3)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> aList = [2, 1, 3]
>>> aList.sort()
>>> aList
[1, 2, 3]
```

Closing for now, but interested in some evidence of where this doesn't work...

**Comment by cjw296** [comment]
Now I see it, aList evidently isn't a list:

```
Traceback (most recent call last):
  File "/Users/chris/vcs/git/xlutils/xlutils/tests/test_filter.py", line 530, in test_finish_resets
    f.finish()
  File "/Users/chris/vcs/git/xlutils/xlutils/filter.py", line 810, in finish
    BaseWriter.finish(self)
  File "/Users/chris/vcs/git/xlutils/xlutils/filter.py", line 595, in finish
    self.close()
  File "/Users/chris/vcs/git/xlutils/xlutils/filter.py", line 268, in close
    self.wtbook.save(stream)
  File "/Users/chris/vcs/git/xlwt/xlwt/Workbook.py", line 696, in save
    doc.save(filename_or_stream, self.get_biff_data())
  File "/Users/chris/vcs/git/xlwt/xlwt/Workbook.py", line 653, in get_biff_data
    before += self.__all_fonts_num_formats_xf_styles_rec()
  File "/Users/chris/vcs/git/xlwt/xlwt/Workbook.py", line 556, in __all_fonts_num_formats_xf_styles_rec
    return self.__styles.get_biff_data()
  File "/Users/chris/vcs/git/xlwt/xlwt/Style.py", line 184, in get_biff_data
    result += self._all_fonts()
  File "/Users/chris/vcs/git/xlwt/xlwt/Style.py", line 196, in _all_fonts
    alist.sort()
AttributeError: 'dict_items' object has no attribute 'sort'
```

**Comment by cjw296** [comment]
Fixed by 59504352ad7ed9573de430a4239fb580c765b162
